src/transformers/configuration_fsmt.py

```diff
@@ -126,9 +126,9 @@ class FSMTConfig(PretrainedConfig):
     # update the defaults from config file
     def __init__(
         self,
-        langs,
-        src_vocab_size,
-        tgt_vocab_size,
+        langs=["en", "de"],
```

Comment:
IMO, it should be possible to instantiate every config (which is not a "composition" config) without providing any parameters: 

```python 
config = self.config_cls()
```

I added these init params from: https://huggingface.co/facebook/wmt19-en-de (cc @stas00)

Reply:
These particular defaults make no sense w/o the corresponding vocab. Just as well set them to 1000 if defaults are now required. I remember some common tests fails if vocab size is less than 1000, so that is probably a good default.

Reply:
I'd say, in such case let's use `["xx", "yy"]` so it's clear it's non-sense. Using meaningful data here is misleading at best.

Reply:
But it's the actual `configuration_fsmt.py` file no? All other configuration files use the actual params of one of the main models as default. E.g. `BertConfig()` uses the params of `bert-base-cased` as defaults. I don't really see why this is misleading... @sgugger @LysandreJik what is your opinion here?